assets/model_monitoring/components/generation_safety_quality/annotation_compute_histogram/spec.yaml

```diff
@@ -66,6 +66,24 @@ outputs:
   histogram:
     type: mltable
     mode: direct
+  samples_index:
+    type: mltable
+    mode: direct
+  groundedness_violations:
```

Comment:
Did you consider just "violations" and then mark the kind of violation in the output data? Just wondering whether this kind of explosion of outputs makes sense for every possible metric.

Reply:
yeah right now it needs to be like this (ie one table for every metric) for the UI to consume